BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -279,7 +286,11 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
   }
   
   @objc private func tappedSendTip() {
-    let controller = TippingViewController(state: state, publisherId: "") // TODO: pass publisher Id
+    // TODO: Figure out state when there's no publisher
+    // Not sure what to do if there's no publisher info..
+    // After all, you need their ID to send a tip..
+    guard let publisherInfo = self.publisherInfo else { return }
+    let controller = TippingViewController(state: state, publisherInfo: publisherInfo)
```

Comment:
Honestly I'd say its probably better to just re-query the publisher info via `state.ledger.publisherInfo`, as when a publishers exclusion state changes it won't be changed in the local variable

Reply:
Fixed.